.github/workflows/core.yml

```diff
@@ -18,11 +18,15 @@ jobs:
       - name: Setup
         uses: actions/setup-node@v2
         with:
-          node-version: '16'
+          node-version: '16.13.1'
           cache: 'npm'
           cache-dependency-path: core/package-lock.json
       - name: Install and Build
         run: |
+          npm run prebuild:tokens && npm run build:tokens
```

Comment:
I don't quite understand why it is not working without the linking of the projects. And while this is a solution, I think, it is wrong to do it. Because, we can now build a version of core, that relies on the linked tokens or icons, even if those are not yet published. And the build will succeed. IMO, the build should use fixed versions of tokens & icons and not rely on npm link